menas/ui/components/schema/schemaDetail.controller.js

```diff
@@ -109,6 +110,17 @@ sap.ui.define([
         })
     },
 
+    _getFileExtension: function (mimeType) {
```

Comment:
Seems like the cob isn't added there
![image](https://user-images.githubusercontent.com/17467768/60261513-5d79c800-98dc-11e9-8457-4418eb24d2b3.png)

I am uploading a copybook that is `NAMEXX.txt` and mongo says `{ "fileMIMEType": "text/plain" }`

Reply:
Actually, even JSON now downloads a txt

Reply:
Mine have `.json` going in and `.json` coming out. Everything that has `.txt` comes out `.txt`. Things that have `.cob` or no extension going in, come out `.cob`. I'm not in control of the MIME type coming in...
What is your suggestion?

Reply:
Not a suggestion but an issue 
Everything downloads as plain text file

Reply:
It happens, thanks for testing!